plugins/module_utils/mso.py

```diff
@@ -801,7 +801,7 @@ def query_objs(self, path, key=None, api_version="v1", **kwargs):
         found = []
         objs = self.request(path, api_version=api_version, method="GET")
 
-        if objs == {} or objs == []:
+        if not objs:
             return found
 
         if key is None:
```

Comment:
This change will impact every modules, did we run all tests for all modules (not just the one modified) as part of this change tests?

Reply:
I tested only with the modified files. Since the Remove cloud site PR has not yet been merged, I was unable to run the other module tests to verify the actual impact.

When the L3Out or tenant template is entirely empty, the latest NDO returns `None` rather than `[]` or `{}`. To manage the `None` value, I adjusted the if condition. The current condition prevents the function from executing for any type of empty value.